redfish-exporter/README.md

```diff
@@ -1,158 +1,87 @@
-# README #
+# AMD Redfish Exporter
 
-The AMD Redfish Exporter is a Go server that subscribes to events and telemetry from a server with Redfish service enabled. It then exposes this data on a Prometheus-compatible metrics route for scraping. Additionally, it can initiate specific Slurm actions in response to certain events.
+The AMD Redfish Exporter is a Go server that subscribes to events and telemetry from servers with Redfish service enabled. It exposes this data on a Prometheus-compatible metrics route for scraping. The exporter also integrates with SLURM to drain nodes when critical hardware events are detected.
 
-## Getting Started
+## Features
 
-### Prerequisites
-- Docker
-- Go 1.23+
+- Subscribes to Redfish events from multiple servers
+- Processes and filters events based on configurable criteria
+- Exposes Prometheus-compatible metrics
+- Integrates with SLURM for node management
+- Supports SSL/TLS for secure communication
 
-### Running the Redfish Mock Server Locally
-To test the exporter locally, you can deploy a Redfish mock server using Docker:
-```bash
-docker run -d --net="host" --name redfish-mock dmtf/redfish-mockup-server:latest --port=8000
-```
+## Installation
 
-> [!IMPORTANT]
-> The `--net="host"` option allows the containerized application to share the host's networking, facilitating access to the listener endpoint. This configuration is essential if the applications aren't on the same network (e.g., when the listener app runs on the host).
-
-#### Testing a Subscription
-To manually create a subscription, create a JSON file (e.g. `data.json`) with the following content:
-```json
-{
-    "Destination":"http://localhost:8080/",
-    "Protocol":"Redfish",
-    "EventFormatType":"Event",
-    "IncludeOriginOfCondition":"true",
-    "EventTypes":[
-       "Alert",
-       "ResourceRemoved",
-       "ResourceAdded",
-       "ResourceUpdated",
-       "StatusChange"
-    ]
- }
-```
+### Prerequisites
 
-Send a POST request to register this subscription:
-```bash
-curl -d "@data.json" -X POST http://localhost:8000/redfish/v1/EventService/Subscriptions
-```
+- Go 1.23 or later
```

Comment:
Docker?

Reply:
The reason is that the make command will run a Docker in a this step.
https://github.com/nod-ai/ADA/blob/70bfacd599fa440b52750b00aa2184b84621eb1b/redfish-exporter/api/Makefile#L4

Reply:
Good call. Added Docker to the prerequisites in the readme and user guide